packages/crypto/lib/models/wallet.js

```diff
@@ -165,7 +165,7 @@ module.exports = class Wallet {
       return false
     }
 
-    const keysgroup = multisignature.keysgroup.map(publicKey => {
+    const keysgroup = transaction.asset.multisignature.keysgroup.map(publicKey => {
```

Comment:
Have you tested this after the multisig registration tx? This line will look for `transaction.asset.multisignature.keysgroup` in every transaction following the registration. It won't exist in anything but the Type 4 tx.

Reply:
I recommend running a testnet and running `packages/core-tester-cli/bin/tester multi-signatures`